lib/routes.js

```diff
@@ -287,7 +292,9 @@ module.exports = async function (app, opts) {
       ))
     } else {
       // Regular query
-      return execute(request.body, request, reply)
+      const data = execute(request.body, request, reply)
+
+      return data
```

Comment:
I think this change is not needed.

Reply:
Sorry, missed restoring this from my testing, fixed it now!